tests/E2E/test/campaigns/common_scenarios/contact.js

```diff
@@ -49,7 +54,8 @@ module.exports = {
       test('should click on "Edit" button', () => {
         return promise
           .then(() => client.waitForExistAndClick(Contact.Contacts.edit_button))
-          .then(() => client.getParamFromURL('id_contact', 1000));
+          .then(() => client.getURL(2000))
+          .then((res) => global.param['id_contact'] = /contacts\/(\d+)\//g.exec(res.value)[1]);
```

Comment:
@boubkerbribri We'll try to improve this in order to make it easy for you guys to retrieve this param.
Do you want us to open an issue "Improve how E2E tests extract data from migrated pages" ? We could have a "top 5" of the worst tricks you had to use, and we'll try to remove them all

Reply:
Thanks @matks , we will look into that in the migration(puppeteer) project, and I hope that we will not be using this way to get id_contact, otherwise we will create an issue.